plugins/config-file.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: config-file
+spec:
+  version: "v0.1.0"       
+  platforms:
+  - selector:             
+      matchExpressions:
+      - {key: os, operator: In, values: [darwin, linux]}
+    uri: https://github.com/Modulus/kubectl-cfg/archive/0.0.6.zip
+    sha256: "325aa06035e07d6da42ee5c0e44b29b92990b3584589447dcf80b093114b0a43"
+    files:
+    - from: "/kubectl-config-file*/*.sh"
+      to: "."              
+    bin: "./kubectl-config-file.sh"  
+  shortDescription: List available kubeconfigs and link them to ~/.kube/config.
+  homepage: https://github.com/Modulus/kubectl-config-file
+  description: |
+    List available kubeconfigs and link them to ~/.kube/config. So you can work
```

Comment:
Just curious: What will your plugin do, if `~/.kube/config` already exists and is not a link?
(No need for action)

Reply:
It will create the link. Meaning you loose the content

Reply:
I have added a new release and added the correct hash and zip file for the current version

Reply:
I have Signed the CLA